modules/panorama/main.tf

```diff
@@ -15,14 +15,14 @@ resource "azurerm_network_interface" "this" {
   name                 = var.interface[0].name
   location             = var.location
   resource_group_name  = var.resource_group_name
-  enable_ip_forwarding = lookup(var.interface[0], "enable_ip_forwarding", "false")
+  enable_ip_forwarding = lookup(var.interface[0], "enable_ip_forwarding", false)
 
   ip_configuration {
     name                          = var.interface[0].name
     subnet_id                     = var.interface[0].subnet_id
     private_ip_address_allocation = lookup(var.interface[0], "private_ip_address", null) != null ? "static" : "dynamic"
     private_ip_address            = lookup(var.interface[0], "private_ip_address", null) != null ? var.interface[0].private_ip_address : null
-    public_ip_address_id          = lookup(var.interface[0], "public_ip", "false") != "false" ? try(azurerm_public_ip.this[0].id) : null
```

Comment:
The `try(1)` is exactly the same as `1`. For the try() to do anything,
it needs to have two or more arguments: `try(1, null)` or `try(1, 2, 3,
4)`